docs/Serverless.md

```diff
@@ -324,3 +325,127 @@ export default async (req, res) => {
     app.server.emit('request', req, res);
 }
 ```
+
+## Google Cloud Functions
+
+You can easly use Fastify inside your Google Cloud Functions.
+
+### Creation of Fastify instance
+```js
+const fastify = require("fastify")({
+  logger: true // you can also define the level passing an object configuration to logger: {level: 'debug'}
+});
+```
+
+### Add Custom `contentTypeParser` to Fastyfy instance
+
+As explained [here](https://github.com/fastify/fastify/issues/946#issuecomment-766319521), since Google Cloud Functions platform perform the parsing of the body request before it arrives into Fastify instance, troubling the body request in case of `POST` and `PATCH` methods, you need to add a custom [`ContentTypeParser`](https://www.fastify.io/docs/latest/ContentTypeParser/) to mitigate this behavior.
```

Comment:
```suggestion
As explained [here](https://github.com/fastify/fastify/issues/946#issuecomment-766319521), since the Google Cloud Functions platform parses the body of the request before it arrives into Fastify instance, troubling the body request in case of `POST` and `PATCH` methods, you need to add a custom [`ContentTypeParser`](https://www.fastify.io/docs/latest/ContentTypeParser/) to mitigate this behavior.
```

Reply:
Thank you @Fdawgs 
Now is much better... :-)